modules/user_manual/pages/apps/admin_audit.adoc

```diff
@@ -0,0 +1,198 @@
+= The Admin Audit App
+:toc: right
+:toclevels: 1
+:leveloffset: +1
+
+== Introduction
+
+The Admin Audit app is an auditing module for ownCloud to trace the actions of users and administrators.
```

Comment:
You need to mention that the admin audit app is enterprise only (https://marketplace.owncloud.com/search/audit)
and therefore needs to be relocated into the enterprise section